fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/api/event/client/HudRenderCallback.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.fabric.api.event.client;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface HudRenderCallback {
+	/** Called after rendering the status effects overlay when rendering the hud. */
+	public static final Event<HudRenderCallback> EVENT = EventFactory.createArrayBacked(HudRenderCallback.class,
+		listeners -> delta -> {
+			for (HudRenderCallback event : listeners) {
+				event.render(delta);
+			}
+		}
+	);
+
+	/**
+	 * Called when the hud is rendered, which is displayed in game, in a world.
+	 *
+	 * @param delta the number of ticks passed since the last tick
```

Comment:
`the number of ticks passed since the last tick` that doesnt sound right.

Reply:
Does `the difference between frames and ticks’ sound better? Sorry I don’t really know how to word it

Reply:
How about `The number of frames passed since the last full tick`?

Reply:
That sounded way better thanks

Reply:
Resolved